logprep/processor/clusterer/processor.py

```diff
@@ -19,34 +23,57 @@
 class Clusterer(RuleBasedProcessor):
     """Cluster log events using a heuristic."""
 
-    def __init__(self, name: str, tree_config: str, logger: Logger,
-                 output_field_name='cluster_signature'):
+    def __init__(self, name: str, logger: Logger, **configuration):
+        tree_config = configuration.get("tree_config")
+        specific_rules_dirs = configuration.get("specific_rules")
+        generic_rules_dirs = configuration.get("generic_rules")
         super().__init__(name, tree_config, logger)
         self.ps = ProcessorStats()
 
         self._name = name
-        self._rules = []
+        self._events_processed = 0
```

Comment:
The events_processed here can be removed as this statistic is tracked in the ProcessorStats object.